data/registry/instrumentation-dotnet-nservicebus.yml

```
@@ -0,0 +1,22 @@
+title: NServiceBus OpenTelemetry Integration # the name of your project
+registryType: instrumentation # the type of integration; is this an exporter, plugin, API package, or something else?
+language: dotnet
+tags:
+  - nservicebus
+  - dotnet
+urls:
+  website: https://particular.net/ # to work on the integrations page as well
+  repo: https://github.com/Particular/NServiceBus
+  docs: https://docs.particular.net/nservicebus/operations/opentelemetry
+license: Reciprocal Public License 1.5
+description: With NServiceBus OpenTelemetry instrumentation enabled, tracing, metrics, and logging can be individually configured via the OpenTelemetry API itself.
+authors:
+  - name: Particular Software
+  - URL: https://github.com/Particular
```

```suggestion
  - name: Particular Software
    url: https://github.com/Particular
```

Please remove the dash, otherwise this are 2 items in the list of authors